arch/arm64/src/common/arm64_vector_table.S

```diff
@@ -78,6 +78,10 @@
 #endif
     stp    x0, x1,  [sp, #8 * REG_ELR]
 
+    mrs    x0, sp_el0
+    mrs    x1, tpidrro_el0
+    stp    x0, x1,  [sp, #8 * REG_SP_EL0]
+
```

Comment:
This patch is completely useless. Kernel doesn't use sp_el0, you are only suppposed to store it when changing context. Like it was done before this PR.